Cooper/Repository/UsersConnection/UsersConnectionRepository.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cooper.Models;
+using Cooper.DAO.Models;
+using Cooper.DAO;
+using Cooper.Repository.Mapping;
+using Cooper.Configuration;
+using Cooper.Models.UserConnectionsEnumTypes;
+
+namespace Cooper.Repository
+{
+    public class UsersConnectionRepository : IUsersConnectionRepository
+    {
+        private IUsersConnectionDAO userConnectionsDAO;
+        private ModelsMapper mapper;
+        
+
+        public UsersConnectionRepository(IConfigProvider configProvider)
+        {
+            userConnectionsDAO = new UsersConnectionDAO(configProvider);
+            mapper = new ModelsMapper();
+        }
+        public List<User> GetSpecifiedTypeUsersList(long userId, ConnectionType specifiedType)
+        {
+            List<UserDb> users = userConnectionsDAO.GetSpecifiedTypeUsersList(userId, specifiedType);
+            List<User> users_newTyped = new List<User>();
+
+            if (users != null)
+            {
+                foreach (var user in users)
```

Comment:
Can be replaced with
users_newTyped = users.Select(mapper.Map).ToList();

Reply:
Yeep)